products/jbrowse-cli/src/base.ts

```diff
@@ -49,67 +49,29 @@ export default abstract class JBrowseCommand extends Command {
   }
 
   async readJsonConfig(location: string) {
-    let locationUrl: URL | undefined
-    try {
-      locationUrl = new URL(location)
-    } catch (error) {
-      // ignore
-    }
-    if (locationUrl) {
-      const response = await fetch(locationUrl)
-      if (response.ok) {
-        return response.json()
-      }
-      throw new Error(`${response.statusText}`)
-    }
     return fsPromises.readFile(location, { encoding: 'utf8' })
   }
 
   async resolveFileLocation(location: string, check = true, warning = false) {
     let locationUrl: URL | undefined
-    let locationPath: string | undefined
     try {
       locationUrl = new URL(location)
     } catch (error) {
       // ignore
     }
     if (locationUrl) {
-      let response
-      try {
-        if (check) {
-          response = await fetch(locationUrl, { method: 'HEAD' })
-          if (response.ok) {
-            return locationUrl.href
-          }
+      if (check) {
+        const response = await fetch(locationUrl, { method: 'HEAD' })
+        if (!response.ok) {
           throw new Error(`${response.statusText}`)
-        } else {
-          return locationUrl.href
         }
-      } catch (error) {
-        // ignore
-      }
-    }
-    try {
-      if (check) {
-        locationPath = await fsPromises.realpath(location)
-      } else {
-        locationPath = location
+        return locationUrl.href
       }
-    } catch (e) {
-      // ignore
+      return locationUrl.href
     }
-    if (locationPath) {
-      const filePath = path.relative(process.cwd(), locationPath)
-      if (warning && filePath.startsWith('..')) {
-        this.warn(
-          `Location ${filePath} is not in the JBrowse directory. Make sure it is still in your server directory.`,
-        )
-      }
-      return filePath
```

Comment:
Need to replace this warning so the `warning` param to this method is used.

Reply:
What if we remove the warning param? It  appears unused to me

Reply:
The warning param is unused because this PR removes the code that uses it. I think that could should remain.

Reply:
I did not see any code paths that use it. I also don't understand the warning?

Reply:
It refers to both JBrowse directory and server directory

Reply:
`warning` is used here:

https://github.com/GMOD/jbrowse-components/blob/c6ef404418477129efde7885793ebb75710d1135/products/jbrowse-cli/src/base.ts#L103-L107

The idea is that if you're using a local file, using `--load trust`, your server root is `/var/www/html`, your JBrowse is at e.g. `/var/www/html/jbrowse` and you try to add a track that is not in your JBrowse directory, this will warn you that you need to make sure it's still in your server directory (e.g. in this case `../data/my.bam` would be fine, but `../../../data/my.bam` or `/data/my.bam` would not.)

Reply:
If I'm using --load trust, I need the tool to just trust me on this....I would prefer no warn?

Reply:
This is exactly what happened to Shihab, he tried to use `--load trust` with files in his `/data/` directory. If this warning was working properly, it could have helped him figure out what was wrong.

I agree it's a confusing term, though. I'd prefer `load --inPlace`.

Reply:
To me, trust means we gotta trust the user. If they misuse it, that is their perogitive. If load trust really is confusing, we could remove it and make the user go through an entire different path

I want the tool to believe that what I'm doing is right without adding combinations of --force, --skipCheck, etc to make it do what I want it to.